company.html

```diff
@@ -0,0 +1,77 @@
+<div class="companyBox"  ng-controller="companyCtrl" >
+
+    <div class="searchBox">
+        <span>Wyszukaj firmę: </span>
+        <input class="inputSearch" ng-model="search.nazwa" placeholder="Nawa firmy">
+        <input class="inputSearch" ng-model="search.adres" placeholder="Adres firmy">
+        <input class="inputSearch" ng-model="search.nip" placeholder="Nip firmy">
+        <input class="inputSearch" ng-model="search.nr_konta" placeholder="Numer konta">
+        <span class="addCompany"><button class="buttonAddCompany" ng-click="addDataCompany()">Dodaj firmę</button></span>
+    </div>
+
+    <table>
+        <thead>
+            <th class="thSort">
+                <button  ng-click="sortBy('nazwa')">Nazwa firmy <span class="sortorder" ng-show="propertyName === 'nazwa'" ng-class="{reverse: reverse}"></span></button>
```

Comment:
aaa widze ze to nie dlatego ze twoj kod tak ma, tylko sama deklaracja klasy "company" ma polskie pola. No to tym bardziej zle, absolutnie tak nie rob :) Wszedzie angielski :)